numba/cgutils.py

```diff
@@ -559,18 +559,28 @@ def cm_cond(cond, inner_cm):
 
 
 @contextmanager
-def loop_nest(builder, shape, intp):
+def loop_nest(builder, shape, intp, inverted=False):
```

Comment:
Instead of *inverted*, wouldn't it be clearer if it is called *order* where the args can be `C` or `F`?